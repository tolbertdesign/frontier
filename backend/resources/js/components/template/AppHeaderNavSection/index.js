import AppHeaderNavSection from './AppHeaderNavSection.vue'

export default AppHeaderNavSection
