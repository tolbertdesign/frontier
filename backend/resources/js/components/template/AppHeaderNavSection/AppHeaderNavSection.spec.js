import { shallowMount, createLocalVue } from '@vue/test-utils'
import AppHeaderNavSection from './AppHeaderNavSection.vue'

const localVue = createLocalVue()

describe('AppHeaderNavSection', () => {
  it('Snapshop test for AppHeaderNavSection', () => {
    const wrapper = shallowMount(AppHeaderNavSection, {
      localVue,
      slots: {
        default: '<div class="slot"></div>',
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
