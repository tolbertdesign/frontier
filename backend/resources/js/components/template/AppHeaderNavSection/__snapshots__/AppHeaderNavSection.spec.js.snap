// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeaderNavSection Snapshop test for AppHeaderNavSection 1`] = `
<div class="nav-section m-0 p-2 px-4 text-grey-darker font-bold border-secondary bg-grey-light">
  <div class="slot"></div>
</div>
`;
